.github/workflows/check-pr.yml

```diff
@@ -51,6 +51,81 @@ jobs:
         with:
           image: ghcr.io/podkrepi-bg/api/migrations:pr
 
+  build-migrations-docker-image:
+    name: Build migrations docker image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Login to GitHub Packages Docker Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get tag
+        if: ${{ startsWith(github.ref, 'refs/tags/v') }}
+        run: echo "VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+
+      - name: Get sha
+        if: ${{ !startsWith(github.ref, 'refs/tags/v') }}
+        run: echo "VERSION=master-$(echo $GITHUB_SHA | cut -c 1-6)" >> $GITHUB_ENV
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        env:
+          NODE_ENV: production
+        with:
```

Comment:
actually the migrations container is built in the previous step. Check line 38

Reply:
this was only temporary from the release.yml for testing the build, removed now. If anything needs to be fixed it needs to be updated in release.yml